include/mbedtls/x509_crt.h

```diff
@@ -97,6 +99,44 @@ typedef struct mbedtls_x509_crt
 }
 mbedtls_x509_crt;
 
+/*
+ * From RFC 5280 section 4.2.1.6:
+ * OtherName ::= SEQUENCE {
+ *      type-id    OBJECT IDENTIFIER,
+ *      value      [0] EXPLICIT ANY DEFINED BY type-id }
+ */
+typedef struct mbedtls_x509_san_other_name
```

Comment:
Minor: Should this be part of the CRT 'namespace', i.e. be named `mbedtls_x509_crt_san_other_name`?

Reply:
perhaps, but wouldn't this be used in csr as well?